components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -90,6 +90,11 @@ export async function findUnstoppableDomainAddress (address: string) {
   return apiProxy.ethJsonRpcController.unstoppableDomainsGetEthAddr(address)
 }
 
+export async function findERCTokenInfo (contractAddress: string) {
```

Comment:
We should call it `getERCTokenInfo` to avoid confusion with other `find*` functions.

Reply:
Can you also please add the return type here?

Reply:
Updated!